lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -39,18 +92,53 @@ describe('NxBinaryDonutChart', function() {
     expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
   });
 
-  it('renders only a background circle path percent is negative', function() {
-    const circles = getShallowComponent({ percent: -50}).find('circle');
-    const paths = getShallowComponent({ percent: -50}).find('path');
+  it('renders the background path and an arc circle with the expected properties when calculated percent is 100',
+      function() {
+        const circles = propsComponent({value: 100}).find('circle');
+        const paths = propsComponent({value: 100}).find('path');
+
+        expect(circles.length).toBe(1);
+        expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
+        expect(circles.at(0)).toHaveProp('strokeWidth', 7);
+        expect(circles.at(0)).toHaveProp('r', 10.5);
+        expect(paths.length).toBe(1);
+        expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+      });
+
+  it('renders only a background circle path when percent is negative', function() {
+    const circles = DeprecatedPropsComponent({percent: -50}).find('circle');
+    const paths = DeprecatedPropsComponent({percent: -50}).find('path');
+
+    expect(circles.length).toBe(0);
+    expect(paths.length).toBe(1);
+    expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+  });
+
+  it('renders only a background circle path when calculated percent is negative', function() {
+    const circles = propsComponent({value: -50}).find('circle');
+    const paths = propsComponent({value: -50}).find('path');
 
     expect(circles.length).toBe(0);
     expect(paths.length).toBe(1);
     expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
   });
 
   it('renders a background path and arc path with expected properties when 0 < percent < 100', function() {
-    const circles = getShallowComponent({ percent: 50}).find('circle');
-    const paths = getShallowComponent({ percent: 50}).find('path');
+    const circles = DeprecatedPropsComponent({percent: 50}).find('circle');
+    const paths = DeprecatedPropsComponent({percent: 50}).find('path');
+
+    expect(circles.length).toBe(0);
+    expect(paths.length).toBe(2);
+    expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    expect(paths.at(1)).toHaveClassName('.nx-binary-donut-chart__arc');
+    expect(paths.at(1)).toHaveProp('d', 'M 0 -10.5 A 10.5 10.5 0 0 1 ' + (-10.5 * Math.cos(1.5 * Math.PI))
+        + ' ' + (-10.5 * Math.sin(1.5 * Math.PI)));
+    expect(paths.at(1)).toHaveProp('strokeWidth', 7);
+  });
+
+  it('renders a background path and arc path with expected properties when 0 < calculated percent < 100', function() {
+    const circles = propsComponent({value: 50}).find('circle');
+    const paths = propsComponent({value: 50}).find('path');
```

Comment:
A lot of these `value` tests should really check the case with a non-default `maxVal` as well

Reply:
This is updated here 2b591aae733e6fe31f0544d52ab760c662ce85cf
There are two describe blocks added for the use of `value` and `percent` respectively. In each block, there are unit tests for whether the maxVal is specified or not.
Since we do have test for percent between 0 and 100 `'renders a background path and arc path with expected properties when 0 < percent < 100'`, I think the following two tests are repeated, so I deleted them.
- renders an arc circle and its background with the expected properties when percent is 72
- renders an arc circle and its background with the expected properties when percent is 11